subconscious/model.py

```diff
@@ -265,8 +265,19 @@ def get_index_key(cls, column_name):
         return all_keys
 
     @classmethod
-    async def all(cls, db, order_by=None):
-        for redis_key in await cls._get_ids_for_all(db, order_by=order_by):
+    async def all(cls, db, order_by=None, limit=None, offset=None):
+        ids_to_iterate = await cls._get_ids_for_all(db, order_by=order_by)
+        if order_by:
```

Comment:
Ya, that's yucky too. I just pushed a PR to allow it all since it's technically not wrong (just strange).